src/overlays/actors/ovl_En_Karebaba/z_en_karebaba.c

```diff
@@ -9,67 +13,511 @@ void EnKarebaba_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnKarebaba_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnKarebaba_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void EnKarebaba_SetupGrow(EnKarebaba*);
+void EnKarebaba_SetupIdle(EnKarebaba*);
+void EnKarebaba_Grow(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_Idle(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_Awaken(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_Spin(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_Dying(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_DeadItemDrop(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_Retract(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_Dead(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_Regrow(EnKarebaba* this, GlobalContext* globalCtx);
+void EnKarebaba_Upright(EnKarebaba* this, GlobalContext* globalCtx);
+
 const ActorInit En_Karebaba_InitVars = {
     ACTOR_EN_KAREBABA,
     ACTORTYPE_ENEMY,
-    FLAGS,
+    5,
     OBJECT_DEKUBABA,
     sizeof(EnKarebaba),
     (ActorFunc)EnKarebaba_Init,
     (ActorFunc)EnKarebaba_Destroy,
     (ActorFunc)EnKarebaba_Update,
-    (ActorFunc)EnKarebaba_Draw,
+    (ActorFunc)EnKarebaba_Draw
+};
+
+static ColliderCylinderInit sBodyCollider = {
```

Comment:
```suggestion
static ColliderCylinderInit sBodyColliderInit = {
```
Might want to work `Init` in there somehow so we know this is the initialization parameters. Same for the head.